### ISSUE TITLE
Forgero recipes not loading and causing crash of REI

### ISSUE THREAD

**Issue by GameBurrow** [SigmundGranaas/forgero#234]
This mod seems like a neat addition to our modpack, however when I attempted to add it, I noticed I couldn't see any receipes with REI  except assembler and touching that item causes REI to crash. 

When I opened my logs after crash I noticed it was full of forego errors.

[latest.log](https://github.com/SigmundGranaas/forgero/files/10276821/latest.log)

**Comment by SigmundGranaas** [comment]
Hi, and thanks for the report! It seems despite spending hours making sure everything works as intended, I still managed to upload the wrong version to CurseForge. The new version of Forgero uses modules to provide the actual content to Forgero, making it possible to only include the content that you think fits best. Unfortunately, there is still no documentation for how this works, as I am trying to upgrade to 1.19.3 asap. New file uploading to CurseForge now, it should work.

TLDR:
Forgero Complete edition: contains everything below in a single jar, probably the one you want to check out

Forgero: only contains the core of the mod, no content
Forgero-vanilla: contains resources for recreating modular tools with vanilla friendly parts
Forgero-extended: contains additional parts, and materials (Vein mining pickaxes, katanas, and similar), needs vanilla module
Forgero-compat: compatability module for other mods (Redstone ingots and mythicmetals, more coming when they update)
Forgero-fabric-compat: Compatability for other fabric mods: Currently Patchouli and lambDynamiclights
Forgero-structures: contains some new structures, currently uses experimental world settings for 1.19.2, so it shows up as a warning. This is fixed in the 1.19.3 version

Currently, I tried to use the Fabric Datagen api for creating recipes, but it is not suited for my needs, which is why you see the infinite list of errors in the log. I will prioritize generating assets another way, and update the mod, I'll track the issue here #235. I tested the new uploaded version, but could not make it crash when hovering the assembly table, so I hope the issue was caused by wrong version uploaded to Curse. Sorry for the inconvenience!
